functests/performance/topology_manager.go

```diff
@@ -43,8 +52,8 @@ var _ = Describe("[rfe_id:27350][performance]Topology Manager", func() {
 		Expect(ok).To(BeTrue())
 		Expect(enabled).To(BeTrue())
 
-		// verify topology manager poicy
-		Expect(kubeletConfig.TopologyManagerPolicy).To(Equal(kubeletconfigv1beta1.SingleNumaNodeTopologyManager))
+		// verify topology manager policy
+		Expect(kubeletConfig.TopologyManagerPolicy).To(Equal(*profile.Spec.NUMA.TopologyPolicy))
```

Comment:
both NUMA and TopologyPolicy are optional, so better check if both are set, and if not fall back to default (best-effort?), or maybe skip test at all.

Reply:
good point! fixed